test/plugin/test_out_google_cloud_grpc.rb

```diff
@@ -212,33 +217,22 @@ def test_non_integer_timestamp
     use_grpc true
   ).freeze
 
-  @@mock_port = 0 # rubocop:disable Style/ClassVars
```

Comment:
Is this no longer needed because of the changes in this PR, or because the underlying issue has been fixed? If the latter, could we at least split it out into a separate commit?

Reply:
This is no longer needed because we are no longer mocking the RPC server directly. Instead we are mocking out the client library's `LoggingServiceV2Client`'s `write_log_entries` method. `LoggingServiceV2Client` wraps the RPC client stub.